prow/scripts/lib/testing-helpers.sh

```diff
@@ -110,33 +112,39 @@ function waitForTestPodsTermination() {
     return 1
 }
 
-function printImagesWithLatestTag() {
-    retry=10
-    while true; do
-        # shellcheck disable=SC2046
-        images=$(kubectl $(context_arg)  get pods --all-namespaces -o jsonpath="{..image}" |\
-        tr -s '[:space:]' '\n' |\
-        grep ":latest")
+cts::check_crd_exist() {
```

Comment:
question: why this functions are in two files? here and `stability-checker/deploy/stability-checker/testing-helpers.sh`